topics/climate/tutorials/pangeo-notebook/tutorial.md

```diff
@@ -0,0 +1,723 @@
+         -  `coastlines`: we pass a parameter `10m` to get coastlines with a high resolution (non-default);
+         - `set_title` to set a title for each subplot.
+    
+![Customized multi-plot](../../images/CAMS-PM2_5-fc-multi.png)
+
+
+> ### {% icon question %} PM2.5 over Italy
+>
+> Using a Multi-plot between Rome and Naples, can you tell us if the forecasted PM2.5 will increase or decrease during the first 24 hours?
+>
+> > ### {% icon solution %} Solution
+> > We will select a sub-area: 11. East to 15.0 East and 40. N to 43. N. PM2.5 will increase and reach values close to 35 μm.m-3
+> >
+> > > ### {% icon code-in %} Input: Python
+> > > ```python
+> > > fig = plt.figure(1, figsize=[10,10])
+> > > 
+> > > # We're using cartopy to project our data.
+> > > # (see documentation on cartopy)
+> > > proj_plot = ccrs.Mercator()
+> > > 
+> > > # We need to project our data to the new projection and for this we use `transform`.
+> > > # we set the original data projection in transform (here PlateCarree)
+> > > p = dset.sel(time=slice(np.timedelta64(1,'h'),np.timedelta64(1,'D'))).sel(latitude=slice(43., 40.),
+> > >                                                                           longitude=slice(11.,15.))['pm2p5_conc'].plot(transform=ccrs.PlateCarree(),
+> > >                                                                                      vmin = 0, vmax = 35,
+> > >                                                                                      subplot_kws={"projection": proj_plot},
+> > >                                                                                      col='time', col_wrap=4,
+> > >                                                                                      robust=True,
+> > >                                                                                      aspect=dset.dims["longitude"] / dset.dims["latitude"],  # for a sensible figsize
+> > >                                                                                      cmap=cmc.roma_r)
+> > > # We have to set the map's options on all axes
+> > > for ax,i in zip(p.axes.flat,  (np.datetime64('2021-12-22') + dset.time.sel(time=slice(np.timedelta64(0),np.timedelta64(1,'D')))).values):
+> > >     ax.coastlines('10m')
+> > >     ax.set_title("CAMS PM2.5 " + pd.to_datetime(i).strftime("%d %b %H:%S UTC"), fontsize=12)
+> > > # Save your figure
+> > > plt.savefig("CAMS-PM2_5-fc-multi-Italy.png")
+> > > ```
+> > {: .code-in}
+> > 
+> >  ![PM2.5 over Italy](../../images/CAMS-PM2_5-fc-multi-Italy.png)
+> {: .solution }
+{: .question }
+
+
+> ### {% icon comment %}  `latitude=slice(47.3, 36.5)` and not `latitude=slice(36.5, 47.3)`
+> Why did we slice latitudes with `latitude=slice(47.3, 36.5)` and not `latitude=slice(36.5, 47.3)`?
+> - because when using slice, you need to specify values using the same order as in the coordinates. Latitudes are specified in 
+> decreasing order for CAMS.
+>
+{: .comment}
```

Comment:
Perhaps it could be included inside the previous question block?

Reply:
Ok. Yes, it is probably not possible to guess such things. I will look at it more carefully. Thanks.